url.bs

```diff
@@ -2471,8 +2471,25 @@ then runs these steps, returning an <a>ASCII string</a>:
  <li><p>If <var>url</var>'s <a for=url>cannot-be-a-base-URL flag</a> is set, append <var>url</var>'s
  <a for=url>path</a>[0] to <var>output</var>.
 
- <li><p>Otherwise, then <a for=list>for each</a> string in <var>url</var>'s <a for=url>path</a>,
- append U+002F (/) followed by the string to <var>output</var>.
+ <li>
+  <p>Otherwise:
+
+  <ol>
+   <li><p>Let <var>path</var> be a <a for=list>clone</a> of <var>url</var>'s <a for=url>path</a>.
+
+   <li><p>If <var>url</var>'s <a for=url>host</a> is null, <var>path</var>'s <a for=list>size</a> is
+   greater than 1, and <var>path</var>[0] is the empty string, then append U+002F (/) followed by
+   U+002E (.) to <var>output</var> and <a for=list>remove</a> <var>path</var>[0] from
+   <var>path</var>.
```

Comment:
There's no need to remove path[0] from path, consequently no need to create clone of path (see my comment below).